awx/main/utils/common.py

```diff
@@ -774,6 +776,32 @@ def wrap_args_with_proot(args, cwd, **kwargs):
     return new_args
 
 
+def command_args_to_kwargs(arg_list):
```

Comment:
I definitely don't care for this :)

Django already gives us everything we need to parse this - it's how it works when `awx-manage inventory_import` is invoked:

```python
from django.core.management import get_commands, load_command_class

args = ['awx-manage', 'inventory_import', '--inventory-id', '1', '--group-filter', '^.+$', '--host-filter', '^.+$', '--exclude-empty-groups', '--source', u'/tmp/awx_inventory_jj1NQd/tmprMHk5Z', '--custom', '-v1', '--traceback']

command_name = args[1]
command = load_command_class(get_commands()[command_name], command_name)
args = args[2:]
parser = command.create_parser('awx-manage', command_name)
options, _ = parser.parse_args(args)
options = vars(options)
print command
print options
```

Reply:
oh yeah, this was an extremely shoddy component, and I need to replace it. My original thought was that this subset of task class could replace `build_args` with something like `build_command_kwargs`, because I consider it pretty artificial how we construct the list to begin with.

However, I recognize that we should avoid that, just for the sake of being change-averse, and I would prefer to use a route like what you suggest here.

I will return to this and compare to the source code for `call_command`, because I think that we just call the dispatch method, passing in the `options`, and I think this will be 100% identical to what the `call_command` method does, I'll just have to check and test.